webapp/src/app/[locale]/lib/apollo-wrapper.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from '@apollo/experimental-nextjs-app-support/ssr'
 import decodeJWT from 'jwt-decode'
 
-import { graphqlConfig } from 'config'
+import { graphqlConfig } from '../../../config'
```

Comment:
@tetogomez @AngeloCG97 for some reason, during compilation time, `from 'config'` was not recognized as a folder import but instead as a library import so It didn't find the package (of course).